tests/plugins/tabletools/manual/columndeletionerror.md

```diff
@@ -0,0 +1,14 @@
+@bender-tags: 4.7.2, bug, 577
```

Comment:
Tests connected with `tableselection` plugin should be inserted into `tableselection/integrations` directory. It allows to easily find all issues related to integrations between `tableselection` and other plugins.